.upstream-tests/test/std/numerics/complex.number/cmplx.over/conj.pass.cpp

```diff
@@ -14,17 +14,16 @@
 // template<Integral T>   complex<double>      conj(T);
 //                        complex<float>       conj(float);
 
+// Helper to silence warnings where conversions are tested
+#include "../msvc_fixes.h"
+
 #include <cuda/std/complex>
 #include <cuda/std/type_traits>
 #include <cuda/std/cassert>
 
 #include "test_macros.h"
 #include "../cases.h"
 
-#if defined(_MSC_VER)
```

Comment:
> Why has this been moved?

It appears that the pragma needs to occur before `<cuda/std/...>` includes because our code will be the one doing the conversion to double. In user code when complex<int> invokes `pow`, `sqrt`, or other intrinsics that do not have integral overloads we end up producing the warning (correctly).